group4/81960_Victoria_Angelova/practicum_week_5/distributor.cpp

```diff
@@ -0,0 +1,48 @@
+#include "distributor.h"
+
+bool Distributor::assignTask(Task *task, Programmer *programmer)
+{
+    if (programmer->canBeAssigned(task))
+    {
+        programmer->addTask(task);
+        tasksToBeAssigned.remove(task);
+        return true;
+    }
+    return false;
+}
+
+/*void Distributor::addTask(Task* newTask) {

```

Comment:
Do not leave commented code. If it's not needed - remove it. If it's needed fix it.